<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentConfigurationWidget</class>
 <widget class="QWidget" name="AppointmentConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="first_date_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>First date:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="first_date">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fmt: y{1,4}:m{1,2}:d{1,2}</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_date_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Last date:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_date">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fmt: y{1,4}:m{1,2}:d{1,2}</string>
   </property>
  </widget>
  <widget class="QLabel" name="interval_start_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Interval start:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interval_start">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fmt: h{1,2}:m{1,2}:s{1,2}</string>
   </property>
  </widget>
  <widget class="QLabel" name="interval_end_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Interval end:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interval_end">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fmt: h{1,2}:m{1,2}:s{1,2}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_services_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Load services from JSON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_service_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Select from loaded services</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configure_service_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure selected service</string>
   </property>
  </widget>
  <widget class="QLabel" name="estimated_duration_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Estimated duration:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="request_duration_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>191</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Request Estimated Duration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
